packages/providers/inpage-providers-hub/src/floatingButton/index.tsx

```
@@ -0,0 +1,746 @@
+          event.stopPropagation();
+          setIsShowCloseButton(false);
+          showCloseDialog();
+        }}
+      >
+        <svg
+          width="15"
+          height="15"
+          viewBox="0 0 15 15"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)